src/ctia/http/routes/bulk.clj

```diff
@@ -28,7 +39,7 @@
     :ttp            ent/realize-ttp))
 
 (defn create-fn
-  "return the create function provided an entity key name"
+  "return the create function provided an entity type key"
   [k]
   #(write-store k (case k
```

Comment:
I appreciate that this isn't in the current PR, but thought I should comment.
`#()` anonymous functions are great for in place quick calls. But for a function that returns a function I don't think I've ever seen anything used except `(fn []...)`. It also lets you document the parameter for the returned function.

The other approach I'd use would be to use `partial` on `k` and the `create-*` function returned in the case. That does lose the label for the parameter though